weasel/_util/remote_storage.py

```diff
@@ -0,0 +1,190 @@
+import hashlib
+import os
+import site
+import tarfile
+import urllib.parse
+from pathlib import Path
+from typing import TYPE_CHECKING, Dict, List, Optional, Union
+from wasabi import msg
+
+from weasel import __version__
```

Comment:
I would rather do a relative import? I may well be wrong though — perhaps there is a good reason to do it this way?

Reply:
No, it should be relative.